.gitignore

```diff
@@ -2,3 +2,4 @@
 build/
 dist/
 *.egg*/
+.idea/
```

Comment:
I guess this was added by mistake; you should probably ignore this folder from your personal ~/.gitignore` file ;)
Some [reading about gitignore](http://vinyll.scopyleft.fr/using-gitignore-the-right-way/)